Answers.md

```diff
@@ -1,11 +1,12 @@
 # Answers
 
 1. What is React JS and what problems does it try and solve? Support your answer with concepts introduced in class and from your personal research on the web.
-
+it uses JSX, makes sites faster

```

Comment:
Can you elaborate more on how React makes sites faster?
You can check out how React Docs explained this [here](https://reactjs.org/)